perftestnotebook/analyzer.py

```diff
@@ -41,8 +41,23 @@ def get_header(self):
             template_header_content = f.read()
             return template_header_content
 
-    def get_notebook_section(self, func):
+    def get_notebook_section(self, function_name,function_parameter=None):
+        notebook_section = ""
         template_function_folder_path = "testing/resources/notebook-sections/"
-        template_function_file_path = os.path.join(template_function_folder_path, func)
+        template_function_file_path = os.path.join(template_function_folder_path, function_name)
         with open(template_function_file_path, "r") as f:
-            return f.read()
+            line = f.readline()
+            while line:
+                if function_name in line and function_parameter is not None:
+                    parameter_string = ""
+                    for (x,y) in list(function_parameter.items()):
+                        parameter_string += ",{}={}".format(x,y)
+                    close_parethesis_index = line.index(")")
+                    modified_line = (line[:close_parethesis_index] + 
+                                    parameter_string +
+                                    line[close_parethesis_index:])
+                    notebook_section += modified_line
```

Comment:
So it seems like you are modifying the scatter function in the notebook section. I don't think this is a good idea since this method isn't very generalized (it won't work in all cases). It would be cool to have this behaviour, but we should only implement it later on when we have something more general in place.

I suggest doing something different. Add the variables to your notebook section like
```
color = None
alpha = 0
```

Then add those variables to your scatter call in the notebook section file. This way you can control what people might be adding to the scatter function.

Or you can put them into a kwargs dictionary:
```
kwargs = {'color': 'black', 'alpha': 0.3}
plt.scatter(..., **kwargs)
```